README.md

```diff
@@ -13,6 +13,24 @@ The driver is compatible across the entire line of UR robots -- from 3 kg payloa
 Check also [presentations and videos](ur_robot_driver/doc/resources/README.md) about this driver.
 
 
+## Installation
+
+The driver is available as several released packages. On Ubuntu, they can be installed with `apt` by running:
+
+```console
+$ sudo apt-get install ros-${ROS_DISTRO}-ur ros-${ROS_DISTRO}-ur-bringup
```

Comment:
We are currently on the way towards deprecating `ur_bringup`. Therefore, installing `ros-${ROS_DISTRO}-ur` would be sufficient. See #403 for details on that.